packages/fxa-auth-server/config/dev.json

```diff
@@ -115,6 +120,7 @@
           "product_id": "fortressProProduct",
           "product_name": "Fortress Pro",
           "product_metadata": {
+            "capabilities": "fortress",
```

Comment:
At the very least this key needs a `:` after "capabilities"?

Reply:
Not necessarily. The bare `capabilities` key means all clients see this capability when the product is subscribed to, rather than filtering by specific client ID.